### PR TITLE
[New Rule] - Suspicious PsExec Execution

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/133

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/defense_evasion_execution_suspicious_psexesvc.toml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+[metadata]
+creation_date = "2020/08/14"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/14"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies suspicious execution via psexec sysinternal utility by creating a renamed psexec default service name."
```
Comment (brokensound77) [inline rules/windows/defense_evasion_execution_suspicious_psexesvc.toml:9]:
```suggestion
description = "Identifies suspicious psexec activity which is executing from the psexev service that has been renamed, possibly to evade detection."
```

may need re-lint

File: rules/windows/defense_evasion_execution_suspicious_psexesvc.toml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+[metadata]
+creation_date = "2020/08/14"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/14"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies suspicious psexec activity which is executing from the psexev service that has been renamed, possibly to evade detection."
```
Comment (threat-punter) [inline rules/windows/defense_evasion_execution_suspicious_psexesvc.toml:9]:
```suggestion
description = "Identifies suspicious psexec activity which is executing from the psexec service that has been renamed, possibly to evade detection."
```

Should this be psexec? I can't remember what the service name is called off the top of my head.

**Review by threat-punter — APPROVED** [review]

File: rules/windows/defense_evasion_execution_suspicious_psexesvc.toml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+maturity = "production"
+updated_date = "2020/08/14"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies suspicious psexec activity which is executing from the psexec service that has been renamed, possibly to evade detection."
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Suspicious Execution via Psexec Sysinternal Utility"
```
Comment (brokensound77) [inline rules/windows/defense_evasion_execution_suspicious_psexesvc.toml:13]:
```suggestion
name = "Suspicious Process Execution from Renamed psexec Executable"
```

File: rules/windows/defense_evasion_execution_suspicious_psexesvc.toml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+maturity = "production"
+updated_date = "2020/08/14"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies suspicious psexec activity which is executing from the psexec service that has been renamed, possibly to evade detection."
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Suspicious Process Execution from Renamed psexec Executable"
```
Comment (bm11100) [inline rules/windows/defense_evasion_execution_suspicious_psexesvc.toml:13]:
```suggestion
name = "Suspicious Process Execution via Renamed PsExec Executable"
```

**Review by bm11100 — APPROVED** [review]

**Pull request merged**